.github/workflows/update_file.yaml

```diff
@@ -0,0 +1,38 @@
+name: Update files
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '43 1 * * 1'
+
+jobs:
+
+  pull-from-sunbeam:
+    name: Pull files from sunbeam
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout local repository
+        uses: actions/checkout@v4
+
+      - name: set git config
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          git config --global user.email "${GITHUB_ACTOR_ID}+${GITHUB_ACTOR}@users.noreply.github.com"
+          git config --global user.name "$(gh api /users/${GITHUB_ACTOR} | jq .name -r)"
+          git config -l
+          
+      - name: Check pull from sunbeam
+        run: make sync-dashboards-alert-rules
+
+      - name: Create a PR for local changes
```

Comment:
Does this handle the case where there are no new changes?  If there are no changes, it should not create the PR.

Reply:
I have same question here, there since no compare logic exists.